apis/eks/v1alpha1/nodegroup_types.go

```diff
@@ -204,6 +204,12 @@ type NodeGroupScalingConfig struct {
 	MinSize *int64 `json:"minSize,omitempty"`
 }
```

Comment:
Could you add a comment to `DesiredSize` field about what happens if you leave it empty and/or what you should do if you want to use autoscaler so that it appears in documentation?